lib/TInitialState.php

```diff
@@ -124,6 +126,12 @@ protected function publishInitialStateForUser(IUser $user, IRootFolder $rootFold
 			$this->serverConfig->getUserValue($user->getUID(), 'spreed', 'play_sounds', 'yes') === 'yes'
 		);
 
+
+		$this->initialState->provideInitialState(
+			'user_group_ids',
+			$this->groupManager->getUserGroupIds($user)
+		);
```

Comment:
Is there a way to get these information on clients as well?

Reply:
I think the URL is

`GET /ocs/v2.php/cloud/users/{userid}/groups`

Reply:
> Is there a way to get these information on clients as well?

This is also a part of userMetadata, so you already request it with other user info

Reply:
Maybe we just don’t store it yet 🧐 I’ll take a look, thanks to both!